### PR TITLE
Sticky fixes for main content area

### PR DESCRIPTION
**Pull request by bmallred**
Resolves truetandem/e-QIP-prototype#2309
Resolves truetandem/e-QIP-prototype#2161

### CODE REVIEW

File: src/components/Section/Section.jsx [hunk 1]
```diff
@@ -12,20 +12,16 @@ import History from './History'
...
 import Package from './Package'
 import { SectionView, SectionViews } from './SectionView'
 
 class Section extends React.Component {
-
-  /**
-   * Used when routes are updated and render is called for different sections. On initial page load,
-   * the componentDidMount() is rendered. However, subsequent path changes trigger componentWillReceiveProps()
-   */
-  componentWillReceiveProps (updatedProps) {
```
Comment (floresj) [inline src/components/Section/Section.jsx]: Evil componentWillReceiveProps. Get outa heaaa.

**Comment by floresj** [comment]
Sooo much better. Just noticed one thing. So clicking the bottom sectionview buttons works great. However, if you click the left navigation, it doesn't appear to scroll up on my end. One example, if I go to 

`Where you have lived` and scroll to the bottom and then click the next left nav link `Where you went to school`, it doesn't scroll to the top.

selection_684

Then clicking school,
selection_685

**Comment by bmallred** [comment]
Cool I'll check this out

**Comment by bmallred** [comment]
@floresj ready for another go on this one!

**Comment by floresj** [comment]
@bmallred whoohoo!

**Comment by floresj** [comment]
Very nice. Working nicely!

**Pull request merged**